coderd/database/queries/groups.sql

```diff
@@ -22,32 +22,36 @@ LIMIT
 
 -- name: GetGroups :many
 SELECT
-    *
+		sqlc.embed(groups),
+		organizations.name AS organization_name,
+		organizations.display_name AS organization_display_name
 FROM
-    groups
+		groups
+INNER JOIN
+		organizations ON ((groups.organization_id = organizations.id))
 WHERE
-    true
-    AND CASE
-        WHEN @organization_id:: uuid != '00000000-0000-0000-0000-000000000000'::uuid THEN
-            groups.organization_id = @organization_id
-        ELSE true
-    END
-    AND CASE
-        -- Filter to only include groups a user is a member of
-        WHEN @has_member_id::uuid != '00000000-0000-0000-0000-000000000000'::uuid THEN
-            EXISTS (
-                SELECT
-                    1
-                FROM
-                    -- this view handles the 'everyone' group in orgs.
-                    group_members_expanded
-                WHERE
-                    group_members_expanded.group_id = groups.id
-                AND
-                    group_members_expanded.user_id = @has_member_id
-            )
-        ELSE true
-    END
+		true
+		AND CASE
+				WHEN @organization_id:: uuid != '00000000-0000-0000-0000-000000000000'::uuid THEN
+						groups.organization_id = @organization_id
+				ELSE true
+		END
+		AND CASE
+				-- Filter to only include groups a user is a member of
+				WHEN @has_member_id::uuid != '00000000-0000-0000-0000-000000000000'::uuid THEN
+						EXISTS (
+								SELECT
+										1
+								FROM
+										-- this view handles the 'everyone' group in orgs.
+										group_members_expanded
+								WHERE
+										group_members_expanded.group_id = groups.id
+								AND
+										group_members_expanded.user_id = @has_member_id
+						)
+				ELSE true
+		END
```

Comment:
Spacing seems to have gone a bit wack

Reply:
yeah, this was one of the few bits of the file using spaces, so I just converted the whole thing to tabs